docs/Release-Process.md

```diff
@@ -5,44 +5,69 @@ Containers release
+1. The [guest-components release](https://github.com/confidential-containers/guest-components/releases) (or a pinned
+version is picked) and [kata-containers](https://github.com/kata-containers/kata-containers) bumps these versions in
+    [versions.yaml](https://github.com/kata-containers/kata-containers/blob/main/versions.yaml) under
+    `externals.coco-guest-components.version` and the `image-rs` crate in the agent's
+    [`Cargo.toml`](https://github.com/kata-containers/kata-containers/blob/main/src/agent/Cargo.toml).
+    - At this point it makes sense for us to try and bump the guest-components that are built into the peer pod vm image,
+    set by the `git.guest-components.reference` value in [versions.yaml](../src/cloud-api-adaptor/versions.yaml)
+    to match the released versions. For compatibility, we might also need to bump the kata-containers agent to the latest commit
+    hash in our [version.yaml](../src/cloud-api-adaptor/versions.yaml) for testing.
+1. Kata Containers [releases](https://github.com/kata-containers/kata-containers/releases)
+    - Hopefully we've already bumped the guest-components in the previous step, but now we should update:
+      - The kata-containers source branch that we use in [versions.yaml](../src/cloud-api-adaptor/versions.yaml) to
 the tag of the kata-containers release candidate.
-- The `kata-containers/src/runtime` go module that we include in the main `cloud-api-adaptor` [`go.mod`](../src/cloud-api-adaptor/go.mod) and the `csi-wrapper` [`go.mod`](../src/csi-wrapper/go.mod).
-This can be done by running
-    ```
-    go get github.com/kata-containers/kata-containers/src/runtime@<release candidate branch e.g. CCv0>
-    go mod tidy
-    ```
-in the [cloud-api-adaptor](../src/cloud-api-adaptor/) directory and [csi-wrapper](../src/csi-wrapper/) directory.
-
-- The attestation-agent that is built into the peer pod vm image, by updating the `git.guest-components.reference` value in [versions.yaml](../src/cloud-api-adaptor/versions.yaml)
+      - The `kata-containers/src/runtime` go module that we include in the main `cloud-api-adaptor` [`go.mod`](../src/cloud-api-adaptor/go.mod) and the `csi-wrapper` [`go.mod`](../src/csi-wrapper/go.mod). This can be done by running
+        ```
+        go get github.com/kata-containers/kata-containers/src/runtime@<latest release e.g. 3.6.0>
+        go mod tidy
+        ```
+        in the [cloud-api-adaptor](../src/cloud-api-adaptor/) directory and [csi-wrapper](../src/csi-wrapper/) directory.
+1. Trustee releases?
+    - We should bump the `kbs` version in [versions.yaml](../src/cloud-api-adaptor/versions.yaml)
+     to match the released and tested version of trustee, if applicable
+1. cloud-api-adaptor releases with the following phases detailed below:
+    - Pre-release testing
+    - Cutting the release
+    - Post release tasks
+1. The CoCo operator updates to use references to the other component releases and then releases itself
```

Comment:
If the operator release needs to happen after the CAA stuff, we should add something to the release checklist.

Is the step required in the operator for peer pods just bumping the payloads in the remote CRD yaml?

Reply:
So for this release bumping the operator first is fine as we had to roll back the changes to get the peerpod controllers managed by the operator, but if/when we resolve the challenges there we would need to do peer pod before the operator, so we can update the peer pods references like Pradipta did in https://github.com/confidential-containers/operator/pull/362